wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,76 @@
+# Installing and updating drivers 
+In this post, you will learn how to download or update drivers on your computer.
+It is highly recommended to avoid using third-party software to update or install your drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager. You can access the device manager by right-clicking the Windows start button or by pressing <kbd>Win</kbd> + <kbd>X</kbd> and selecting Device Manager. In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Using Windows Update
+You can usually find drivers update for your drivers in Windows Update, instead of manually downloading the file.
+ 1. Open Settings through the start menu or by pressing <kbd>Win</kbd> + <kbd>I</kbd>.
+ 2. Go to Updates and Security or Windows Updates.
+ 3. Click on “Check for Updates”.
+ 4. Windows Update might say that you are up-to-date, even if optional updates are available. In this case, you will see an “Optional Updates” button. Clicking on it will show a list of updates (all of them are typically driver updates). Select all of them and click on “Download and install”.
+
+## Method 2: Download using the manufacturer's website.
+It is highly recommended to download or update your drivers through your manufacturer's website, as that will guarantee proper functionality, provided you've installed the correct drivers. This is specially a good idea for your Graphics Card (GPU) Drivers.
+
+All you need to do is:
+1. On the search engine of your choice (Google, Bing, etc…), search for "{device manufacturer} drivers".
+2. Visit the manufacturer's website and enter the model of your device, then follow the instructions on the page.
+3. When you select the product, you may be shown with 2 options:
+	- Download the driver downloader (recommended).
+	- Download this specific driver only and install manually.
+## Method 3: Download using the OEM's website.
```

Comment:
is this section specifically for GPU only? or is it for all drivers

Reply:
that's my bad, lol made an oopsie in my suggestion and he committed it

Reply:
Then?

Reply:
> Then?

U already fixed that in commit 705b7bc3ce65532d192e87cd890a95d3443f0847